content/components/action-list.mdx

```diff
@@ -188,54 +188,62 @@ Action list items can be selected. Single selections are represented with a [`ch
     </Box>
 </Box>
 
-## Application
-
-### In overlays
+## Examples
 
 <Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 2fr']} gridGap={4}>
     <div>
-        <Link href="https://primer.style/react/ActionMenu">
-            <img alt="Action menu" src="https://user-images.githubusercontent.com/293280/123880964-95b3a700-d8f8-11eb-9775-cbaf165207ed.png" />
+        <Link href="/components/action-menu">
+            <img alt="Action menu" src="https://user-images.githubusercontent.com/980622/234046230-8ae1db9b-bcb9-436c-85f3-b55e02e4f2ac.png" />
         </Link>
     </div>
     <div>
         <Heading sx={{fontSize: 3}}>Action menu</Heading>
         <Box as="p">
+Action menus are a list of items, with each item representing an action, command, or current selection, which can be a single or multi-select.
 
-Action menus are used for disambiguation, navigation, or to display secondary options. They appear when users interact with buttons, actions, or other controls.
-
-[Primer React implementation](https://primer.style/react/ActionMenu)
+[Documentation](/components/action-menu)
```

Comment:
is there a reason for the explicit `Documentation` link, rather than just linking to the component docs from the main heading title?

Reply:
Not sure. It's something we don't do on headings right now in the documentation. I guess because they also don't get underlined.

<img width="1084" alt="Screenshot 2023-04-27 at 14 35 07" src="https://user-images.githubusercontent.com/980622/234878607-424230a1-4732-4cc1-b458-40886d5e3cea.png">